### ISSUE TITLE
Safari test failing with workers and es5

### ISSUE THREAD

**Issue by bentoi** [zeroc-ice/ice#253]
Testing with Safari, web workers and ES5 fail. It works without ES5 however... the following in printed out on the Safari console:
```
[Error] Unhandled Promise Rejection: ReferenceError: Can't find variable: Worker
```
I can reproduce this with Safari 12 on Mojave. It also showed up with distribution testing on Sierra (not on Mojave because Safari testing didn't run ... I'm looking into that one)

**Comment by bentoi** [comment]
ES5 testing also fails with Internet Explorer on Windows (without workers this time):
```
SCRIPT5007: Unable to get property '_require' of undefined or null reference
TestHelper.js (18,2)
SCRIPT5007: Unable to get property '_require' of undefined or null reference
Controller.js (32,2)
SCRIPT5007: Unable to get property '_require' of undefined or null reference
Controller.js (32,2)
...
```

**Comment by pepone** [comment]
I fixed Safari ES5 issue in commit 02914c716c671f7900861e6d611be27afe673cf4

Benoit fixed the IE issue caused by JavaScript tests not being build by the CI on Windows
